fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStore.java

```diff
@@ -2054,18 +2088,37 @@ public Range getUnbuiltRange() {
     }
 
     /**
-     * Loads the current state of the record store asynchronously.
+     * Loads the current state of the record store asynchronously and sets {@code recordStoreState}.
```

Comment:
I don't think the extra docs here are necessary, but maybe I'm wrong.

Reply:
Hm, maybe it would be better then to say that it loads the record store state and caches it in memory so that `getRecordStoreState` requires no i/o. (As right now, the comment references a private member variable.) Clarifying the behavior of `loadRecordStoreState` in `loadRecordStoreState` might also be good.

Reply:
sure.